packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/springboot/configmid.md

```diff
@@ -1,14 +1,22 @@
-Edit `src/main/resources/application.properties` and update these values to include your Okta domain and the Okta Application's client ID and secret:
+The Okta CLI created an `.okta.env` file in your current directory. This file includes your Okta domain, client ID, and client secret:
 
 ```properties
-okta.oauth2.issuer=https://${yourOktaDomain}/oauth2/${authorizationServerId}
-okta.oauth2.client-id=${clientId}
-okta.oauth2.client-secret=${clientSecret}
+OKTA_OAUTH2_ISSUER=https://${yourOktaDomain}/oauth2/${authorizationServerId}
+OKTA_OAUTH2_CLIENT_ID=${clientId}
+OKTA_OAUTH2_CLIENT_SECRET=${clientSecret}
+```
+
+Run the following commands to set these values as environment variables and start your app:
 
-# Customize the callback route path
-okta.oauth2.redirect-uri=/authorization-code/callback
+```shell
+source .okta.env
+./mvnw spring-boot:run
 ```
 
```

Comment:
It might be good to add code to show how to display the user's name after they log in. For example:

```java
@GetMapping("/")
public String hello(Principal principal) {
    return "Hello, " + principal.getName();
}
```